cli/src/commands.rs

```diff
@@ -64,6 +64,12 @@ pub(crate) enum Command {
         /// Delay for the send, in seconds
         #[arg(long)]
         delay: Option<u64>,
+
+        /// Can only be used when a BIP21 address with an amount is provided
```

Comment:
~A normal asset send can be done without a BIP21 address, why not this also?~ Seems this is also works with an asset_id and amount provided